crates/bevy_reflect/derive/src/container_attributes.rs

```diff
@@ -543,6 +557,13 @@ impl ContainerAttributes {
         self.no_field_bounds
     }
 
+    /// Returns true if the `no_auto_register` attribute was found on this type.
+    // This is not feature-gated because derive macro shouldn't break if auto_register feature is disabled.
+    #[allow(dead_code)]
```

Comment:
Could you provide a `reason` for this (i.e. `#[allow(dead_code, reason = "...")]`)?